src/main/java/edu/ucar/unidata/rosetta/controller/TemplateController.java

```diff
@@ -505,13 +734,14 @@ public String getMetadataKnownFile(UploadedAutoconvertFile file, HttpServletRequ
         // "title:title here,description:des here,institution:inst here,dataAuthor:data aut here,version:version here,dataSource:source here"
         return metadataStr;
     }
-
+*/
     /**
      * Accepts a POST request to restore session from an NcML file,
      *
      * @param file The UploadedFile form backing object containing the file.
      * @return A String of the local file name for the ASCII file (or null for an error).
      */
+    /*
     @RequestMapping(value = "/restoreFromZip", method = RequestMethod.POST)
     @ResponseBody
     public String processZip(UploadedFile file) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)